aspnetcore/blazor/fundamentals/signalr.md

```diff
@@ -1127,6 +1127,31 @@ Blazor Server:
 
 **In the preceding example, the `{BLAZOR SCRIPT}` placeholder is the Blazor script path and file name.** For the location of the script and the path to use, see <xref:blazor/project-structure#location-of-the-blazor-script>.
 
+:::moniker range=">= aspnetcore-9.0"
+
+When the user navigates back to an app with a disconnected circuit, reconnection is attempted immediately rather than waiting for the duration of the next reconnect interval. This improves the user experience when navigating to an app in a browser tab that has gone to sleep.
+
+When a reconnection attempt reaches the server but the server has already released the circuit, a page refresh occurs automatically. This prevents the user from having to manually refresh the page if it's likely going to result in a successful reconnection.
```

Comment:
Hm, while these are noteworthy improvements to mention in release notes, I'm not sure if these details are super relevant to this section. Do you think it's fine to leave them out?

Reply:
I think the first one is more relevant to the section, but I see what you mean about the second one.

Do you want me to kill both 🔪 ... or just the second one?

Reply:
I think we can keep the first one, but maybe without the "improves the user experience" bit because "improves" sounds relative to how it worked before, which to me seems more suitable for release notes. But we could still mention the reasoning for it working that way, which is that if the user is actively navigating back to the app, they likely expect things to resume working as quickly as possible (we know they're not sitting idly and twiddling their thumbs).